src/main/zapHomeFiles/hud/utils.js

```diff
@@ -246,7 +243,20 @@ var utils = (function() {
 
 		return Promise.all(promises)
 	}
-	
+
+	/*
+	 * Backs up the frames tools to ZAP
```

Comment:
could we add a little more context in this comment about what this means? Reading "back up" wasn't intuitive to me, so it may be helpful to describe that were persisting configs through multiple loads of the HUD